README.md

```
@@ -7,11 +7,18 @@ Leverages [OpenAI Completions API](https://platform.openai.com/docs/api-referenc
 
 ## Install
 
+>**WARNING**: Incompatible with `ovos-persona<2.0.0`. Stick with `ovos-openai-plugin==2.0.6` if you can't upgrade
+
 `pip install ovos-openai-plugin`
 
 ## Persona Usage
 
 To create your own persona using a OpenAI compatible server create a .json in `~/.config/ovos_persona/llm.json`:  
+
+>**WARNING**: OpenVoiceOS has **deprecated** the solver plugin abstraction. **more info**: https://github.com/OpenVoiceOS/ovos-persona/pull/143
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)